.github/workflows/continous-integration.yml

```diff
@@ -156,26 +162,28 @@ jobs:
 
     - name: Install Dependencies (Windows) 📦
       if: matrix.os == 'windows-latest'
+      # Restoring cache doesn't work properly on Windows due to symlinks.
+      # That's why we're manually removing symlinks here and everything that is inside `$spacy_data_dir` directory.
+      # More information: https://github.com/actions/cache/issues/120
       run: |
-        make install-full-windows
+        $spacy_data_dir = ".venv\lib\site-packages\spacy\data"
+        if (Test-Path $spacy_data_dir) {
```

Comment:
no clue about the Windows scripting, but it looks like we only delete symlinks within `$spacy_data_dir`? The comment suggests that we do that for all of `.venv`

Reply:
👍